### PR TITLE
Fum 3107 - Implement e2e testing

### PR DESCRIPTION
**Pull request by rbapst-tamedia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +39,9 @@ You can configure it by overriding the `scale.minAvailable` parameter. (e.g. `90
 
 ## Tests
 
-There are multiple complementary ways to test the chart before relesing it. We have defined workflow that uses the `chart-testing` (`ct`) library to:
+On PR, a github workflow is run which validates and deploys the app-helm-chart on a k8s cluster using
+various value files (take files ending with `-values.yaml` in the [ci directory](ci)). Workflow is copied from <https://github.com/renovatebot/helm-charts>.
```
Comment (swibrow) [inline README.md]: it takes any .yaml file 
https://github.com/DND-IT/app-helm-chart/pull/47/files#diff-43bbc03c76533ab7bc7c06bfda6378f2622e1b2ad10ff9244eb90f3f495fb97bR34

**Review by swibrow — CHANGES_REQUESTED** [review]
Since kubeval tests are run on each .yaml file in the ci folder, you want to keep the empty one in there so that the chart is also tested with no values file set. In general, a helm chart should at least install without any values set, this is a common requirement for helm chart development.

File: README.md (hunk 1 above)
Comment (rbapst-tamedia) [reply README.md]: Yes for kubeval.sh. but for the `ct install` or `ct lint` (https://github.com/helm/chart-testing/blob/main/doc/ct_lint.md) it uses only "*-values.yaml" files.
I'll update the doc

**Comment by rbapst-tamedia** [comment]
> Since kubeval tests are run on each .yaml file in the ci folder, you want to keep the empty one in there so that the chart is also tested with no values file set. In general, a helm chart should at least install without any values set, this is a common requirement for helm chart development.

added the values-empty.yaml file

File: README.md (hunk 1 above)
Comment (swibrow) [reply README.md]: ah good to know. thanks for the update

**Review by swibrow — APPROVED** [review]
Nice 🚀

**Pull request merged**